src/openforms/forms/models/form_variable.py

```diff
@@ -27,14 +28,27 @@
 from .form_definition import FormDefinition
 
 if TYPE_CHECKING:
+    from rest_framework.request import Request
+
     from .form_step import FormStep
 
 
 def get_now() -> str:
     return timezone.now().isoformat()
 
 
-STATIC_INITIAL_VALUES = {FormVariableStaticInitialValues.now: get_now}
+def get_auth_identifier(request: Optional["Request"]) -> Optional[str]:
+    if not request or not hasattr(request, "session"):
+        return None
+
+    # TODO what is the structure of the data in the machtigen field?
+    return request.session.get(FORM_AUTH_SESSION_KEY)
```

Comment:
I think we should aim to keep django apps decoupled and avoid using the request/session as a persistent data structure:

1. forms app ideally shouldn't have to be aware of `authentication` app internals
2. the request session data is not guaranteed to always be available - in fact, after authenticating, the `authentication` apps is responsible for persisting the auth details asap into the `Submission` model (see `signals.py` there)
3. session data is just a mechanism to persist some information during redirect chains and ideally would be wiped once it's persisted
4. session data is a bit tricky perhaps with the form suspend/resume - it does get set if you re-auth, but it all feels a bit fragile

A submission is essentially a "form session", with the added benefit that it acts as a namespace (you can have different details for different submissions all at the same time). I think it's better to pass around the `submission` instance instead of the request/session and extract the data from there.

On that note, `submissions/authentication` depends on `forms` app, and we did a lot of work to remove `authentication` app imports from submissions/forms - I think it's a bit of a shame that we then re-introduce this tight coupling.

Maybe there should be a method/utility in submissions app `Submission.get_auth_identifier_variable`? Maybe we should have a per-app discovery of variables or a registry for them so that every app can declare their own variables and implementations?